engine/src/page.js

```diff
@@ -1,9 +1,12 @@
 import { merge, reach } from '@hapi/hoek'
 import * as querystring from 'querystring'
+import { redirectTo } from './helpers'
 const joi = require('joi')
 const { proceed } = require('./helpers')
 const { ComponentCollection } = require('../components')
 
+const { RelativeUrl, FeedbackContextInfo, decode } = require('./feedback') /* eslint-disable-line */
```

Comment:
any reason for require vs import?

Reply:
Being consistent with the other stuff in the file. Didn't know if it was ok to mix and match!